questionnaires/vaping_weekly_survey/vaping_weekly_survey_armt.json

```diff
@@ -0,0 +1,856 @@
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "last_two_weeks_matrix",
+        "matrix_ranking": "",
+        "field_annotation": "matrix-radio-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "unusual_lump",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "matrix-radio",
+        "field_label": "Unusual lump or swelling anywhere",
+        "select_choices_or_calculations": [
+            {
+                "code": "1",
+                "label": "Yes "
+            },
+            {
+                "code": "0",
+                "label": "No"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "last_two_weeks_matrix",
+        "matrix_ranking": "",
+        "field_annotation": "matrix-radio-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "unusual_fatigue",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "matrix-radio",
+        "field_label": "Unusual fatigue",
+        "select_choices_or_calculations": [
+            {
+                "code": "1",
+                "label": "Yes "
+            },
+            {
+                "code": "0",
+                "label": "No"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "last_two_weeks_matrix",
+        "matrix_ranking": "",
+        "field_annotation": "matrix-radio-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "smoke_days_10",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "slider",
+        "field_label": "To help people say how good or bad their health state is, we have drawn a scale (rather like a thermometer) on which the best state you can imagine is marked 100 and the worst state you can imagine is marked 0. Please indicate on this scale how good or bad your own health is today, in your opinion.  ",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "Worst imaginable health state "
```

Comment:
The description contains necessary info, can probably just mark this as "Worst" and "Best".

Reply:
Okay